ridgeback_msgs/msg/Status.msg

```diff
@@ -16,12 +16,14 @@ float32 mcu_temperature
 
 # Monitoring the run/stop loop. Changes in these values trigger an immediate
 # publish, outside the ordinarily-scheduled 1Hz updates.
+bool stop_power_status  # True indicates everything is in order with the stop loop.
+bool stop_engaged
 bool drivers_active
-bool driver_external_stop_present
-bool driver_external_stop_stopped
+bool external_stop_present
```

Comment:
We need a little more in the way of comments explaining what these are all about and how they relate to each other.

I know this is based on jackal_msgs, but another source of inspiration could be Grizzly: http://docs.ros.org/indigo/api/grizzly_msgs/html/msg/RawStatus.html